### PR TITLE
Feature/dropdown app bar example

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes PXBLUE-2423 .

#### Changes proposed in this Pull Request:
- Add dropdown toolbar desktop view
- Show bottom sheet for mobile view

#### To Test:
- http://localhost:3000/dropdown-toolbar

### CODE REVIEW

**Comment by huayunh** [comment]
the bottom sheet should be sliding in, rather than fading in

**Comment by huayunh** [comment]
On mobile, this arrow never flipped back
image

**Comment by huayunh** [comment]
On mobile, sometimes the bottomsheet doesn't "slide" in, but rather just fade in.

https://user-images.githubusercontent.com/8997218/133767337-b19ef32d-24ad-4c2e-aeaa-e4722ac20bd4.mov

**Comment by huayunh** [comment]
Log a bug if this is beyond the scope of this PR

**Comment by ektasawant** [comment]
> On mobile, this arrow never flipped back
> image

Fixed please review.

**Comment by ektasawant** [comment]
> Log a bug if this is beyond the scope of this PR

Let me log issue for this.

File: src/pages/app-bar/dropdown-toolbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                        <DropdownToolbar
+                            title={'Appbar Title'}
+                            subtitle={'Dropdown Toolbar'}
+                            menuGroups={menuGroups}
+                        ></DropdownToolbar>
+                    </Hidden>
+                    <Hidden smUp={true}>
+                        <DropdownToolbar
+                            title={'Appbar Title'}
+                            subtitle={'Dropdown Toolbar'}
```
Comment (huayunh) [inline src/pages/app-bar/dropdown-toolbar/index.tsx:72]: In Evan's PR, he had the subtitle changing to whichever item that is currently selected in the dropdown menu. I think it is a good thing — can we do that here as well?

**Review by huayunh — COMMENTED** [review]
I don't know why the app bar got nudged up here... was it just me?

https://user-images.githubusercontent.com/8997218/134044585-354883fb-d3d6-471a-9b84-bbcc1e53cfc9.mov

File: src/pages/app-bar/dropdown-toolbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                            classes={{
+                                subtitleContent: showMenu ? classes.arrowUp : classes.arrowDown,
+                            }}
+                            menu={
+                                <React.Fragment key={'bottom'}>
```
[the remark this answers is not in the document]
Comment (ektasawant) [reply src/pages/app-bar/dropdown-toolbar/index.tsx:77]: Getting below error if I am not using `React.Fragment`.  React is adding `anchorEl` attribute which gives below error.
image

**Comment by ektasawant** [comment]
> I don't know why the app bar got nudged up here... was it just me?
> 
>  PR.mov

Not able to reproduce it.

**Review by huayunh — APPROVED** [review]
All good, just need to fix the unit test

**Pull request merged**